### PR TITLE
cbdc_guide PR 1

### PR DESCRIPTION
**Pull request by Lakshya18jul**

### CODE REVIEW

File: src/central-bank-digital-currency-dodao-academy.yaml [hunk 1]
```diff
@@ -115,31 +111,31 @@ steps:
...
-          - content: D
-            key: db3874d0
-        content: Q3
+          - content: Gold
+            key: A
+          - content: Silver
+            key: B
+          - content: US Dollar
+            key: C
+          - content: Central Bank
```
Comment (RobinNagpal) [inline src/central-bank-digital-currency-dodao-academy.yaml:122]:
```suggestion
          - content: Depends on Central Bank of the country
```

File: src/central-bank-digital-currency-dodao-academy.yaml [hunk 2]
```diff
@@ -191,31 +187,32 @@ steps:
...
-          - content: C
-            key: aff31cd8
-          - content: D
-            key: 2bc62be6
-        content: Q6
+          - content: Cryptocurrency has high value
+            key: A
+          - content: Cryptocurrency is highly fluctuating
+            key: B
+          - content: CBDCs are under government control whereas cryptocurrency cannot be regulated
```
Comment (RobinNagpal) [inline src/central-bank-digital-currency-dodao-academy.yaml:211]:
```suggestion
          - content: CBDCs are under government control whereas cryptocurrency are mostly global and not under specific contry
```

File: src/central-bank-digital-currency-dodao-academy.yaml [hunk 3]
```diff
@@ -430,31 +462,31 @@ steps:
     name: Evaluation
     stepItems:
       - answerKeys:
-          - a5969719
+          - A
```
Comment (RobinNagpal) [inline src/central-bank-digital-currency-dodao-academy.yaml:465]: D is also right?

File: src/central-bank-digital-currency-dodao-academy.yaml [hunk 4]
```diff
@@ -430,31 +462,31 @@ steps:
...
-        content: Q10
+          - content: Backed by Gold
+            key: A
+          - content: Backed by fiat currency
+            key: B
+          - content: Fluctuating value
+            key: C
+          - content: All of the above
+            key: D
+        content: Why CBDCs is better for investors than cryptocurrency?
```
Comment (RobinNagpal) [inline src/central-bank-digital-currency-dodao-academy.yaml:489]:
```suggestion
        content: Why CBDCs is more reliable than cryptocurrency in the current time?
```

File: src/central-bank-digital-currency-dodao-academy.yaml [hunk 5]
```diff
@@ -316,6 +315,39 @@ steps:
...
+            key: D
+        content: What are the most common issues with CBDC?
+        type: SingleChoice
+        uuid: 98dd0750-9388-44c6-86f2-64837758777e
+      - answerKeys:
+          - B          
+        choices:
+          - content: Hacking 
+            key: A
+          - content: Interconversion into CBDC of respective nationalities
```
Comment (RobinNagpal) [inline src/central-bank-digital-currency-dodao-academy.yaml:341]: `Interconversion into CBDC of respective nationalities`  an issue?

File: src/central-bank-digital-currency-dodao-academy.yaml (hunk 1 above)
Comment (Lakshya18jul) [reply src/central-bank-digital-currency-dodao-academy.yaml:122]: Done

File: src/central-bank-digital-currency-dodao-academy.yaml (hunk 2 above)
Comment (Lakshya18jul) [reply src/central-bank-digital-currency-dodao-academy.yaml:211]: Done

File: src/central-bank-digital-currency-dodao-academy.yaml (hunk 5 above)
Comment (Lakshya18jul) [reply src/central-bank-digital-currency-dodao-academy.yaml:341]: I have re phrased it

File: src/central-bank-digital-currency-dodao-academy.yaml [hunk 6]
```diff
@@ -430,31 +462,32 @@ steps:
...
+          - content: Consensus mechanism
+            key: A
+          - content: Decentralised
+            key: B
+          - content: Volatility
+            key: C
+          - content: None of the above
+            key: D
+        content: What makes cryto more secure and transparent compared to cbdc
         type: SingleChoice
```
Comment (RobinNagpal) [inline src/central-bank-digital-currency-dodao-academy.yaml:477]: MultipleChoice

**Review by RobinNagpal — APPROVED** [review]

**Pull request merged**